src/addBusinessDays/test.ts

```diff
@@ -103,11 +103,53 @@ describe('addBusinessDays', function () {
       assert.deepStrictEqual(result, new Date(2022, 0 /* Jan */, 21))
     })
 
-    it('can override business days with exceptions', function () {
+    it('can include business days with true exceptions', function () {
       // Given business days of Monday - Saturday
-      // And exceptions over one weekend
+      // And a true exception on a Sunday
 
-      // When we add 8 business days
+      // When we add 5 business days
+      const result = addBusinessDays(new Date(2022, 1 /* Feb */, 9), 5, {
+        businessDays: [1, 2, 3, 4, 5, 6],
+        exceptions: { '02/13/22': true },
+      })
+
+      // Then we expect to have Sunday 2/13 included as a working day
+      assert.deepStrictEqual(result, new Date(2022, 1 /* Feb */, 14))
+    })
+
+    it('can include business days with true exceptions and ending on the exception', function () {
```

Comment:
question: Do we have any starting on an exception tests?

Reply:
I added a couple